### PR TITLE
explicitly declare and freeze tornado for livereload

### LINKED ISSUE #3961: Fix template autoreload
Some time ago when you used to change a template then the page was refreshed in real time without the need to relaunch the Flask app. Would be nice to restore this functionality, it's very useful especially during development

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug.

Fix #3961.

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [x] tests executed by DN

### CODE REVIEW

**Comment by dnil** [comment]
It doesn't seem to be very maintained. Perhaps it is time to deprecate it as well. I'll do that!

File: scout/commands/serve.py [hunk 1]
```diff
@@ -33,6 +32,9 @@ def serve(host, port, debug, livereload, test):
         sys.exit("No database connection available")
 
     if livereload:
+        LOG.warning(
```
Comment (northwestwitch) [inline scout/commands/serve.py:35]: What about marking the whole serve command as deprecated instead? Like [here](https://github.com/Clinical-Genomics/scout/blob/dd706fba88d0815cb05ad4eea9e93520baf7f2ca/scout/commands/load/report.py#L14)

**Review by northwestwitch — COMMENTED** [review]
The deprecation message will be shown only if serve is runned with the --livereload option, but don't we deprecated the whole thing?

File: scout/commands/serve.py (hunk 1 above)
Comment (dnil) [reply scout/commands/serve.py:35]: Hm, I have a feeling the `serve` command is "nice" for new users/trying stuff out. Perhaps we could even just change it to fire up the app using gunicorn implicitly. But maybe it's just my habit speaking. Let's deprecate it. That way we are free to remove it, but can always change our minds and fix it instead if it seems possible.

**Comment by dnil** [comment]
```
(scout39n) ➜  scout git:(freeze_tornado) ✗ pip install -U -r requirements.txt
...
(scout39n) ➜  scout git:(freeze_tornado) ✗ pip freeze | grep tornado
tornado==6.2
```

Screenshot 2023-05-30 at 16 08 19

Screenshot 2023-05-30 at 16 09 17

File: scout/commands/serve.py [hunk 2]
```diff
@@ -12,7 +11,7 @@
 LOG = logging.getLogger(__name__)
 
 
-@click.command()
+@click.command(deprecated=True)
```
Comment (northwestwitch) [inline scout/commands/serve.py:14]: 👍🏻

**Review by northwestwitch — APPROVED** [review]
Looks good! 💯

**Review by northwestwitch — APPROVED** [review]
Looks good! 💯

**Pull request merged**